scripts/exregional_run_analysis.sh

```diff
@@ -0,0 +1,594 @@
+    fi
+  done
+done
+
+size=`du --apparent-size --block-size=1 --dereference ${ENKF_FCST}/${enkfcstname}.mem001.nemsio | grep -o '..........' | sed -n '1 p'`
+ls ${ENKF_FCST}/${enkfcstname}.mem001.nemsio > filelist03
+
+n=2
+while [[ $n -le 80 ]] ; do
+  
+  if [ $n -lt 10  ]; then
+    nn=00$n
+  else
+    nn=0$n
+  fi
+
+  size2=`du --apparent-size --block-size=1 --dereference ${ENKF_FCST}/${enkfcstname}.mem${nn}.nemsio | grep -o '..........' | sed -n '1 p'`
+
+  if [[ $size2 -lt $size ]]; then
+    print_info_msg "$VERBOSE"  "Bad GDAS member number ${nn}."
+  else
+    ls ${ENKF_FCST}/${enkfcstname}.mem${nn}.nemsio >> filelist03
+  fi
+
+  n=$((n + 1))
+
+done
+
+
+#
+#-----------------------------------------------------------------------
+#
+# set default values for namelist
+#
+#-----------------------------------------------------------------------
+
+cloudanalysistype=0
+ifsatbufr=.false.
+ifsoilnudge=.false.
+beta1_inv=1.0
+ifhyb=.false.
+
+# Determine if hybrid option is available
+memname='atmf009'
+nummem=`more filelist03 | wc -l`
+nummem=$((nummem - 3 ))
+if [[ ${nummem} -eq 80 ]]; then
+  print_info_msg "$VERBOSE" "Do hybrid with ${memname}"
+  beta1_inv=0.15
+  ifhyb=.true.
+  print_info_msg "$VERBOSE" " Cycle ${YYYYMMDDHH}: GSI hybrid uses ${memname} with n_ens=${nummem}" 
+fi
+
+
+#
+#-----------------------------------------------------------------------
+#
+# link or copy background files
+#
+#-----------------------------------------------------------------------
+
+FV3SARPATH=${CYCLE_DIR}
+cp_vrfy ${fixdir}/fv3_akbk                               fv3_akbk
+cp_vrfy ${fixdir}/fv3_grid_spec                          fv3_grid_spec
+
+if [ ${BKTYPE} -eq 1 ]; then  # Use background from INPUT
+  cp_vrfy ${bkpath}/gfs_data.tile7.halo0.nc                gfs_data.tile7.halo0.nc_b
+  ${NCKS} -A -v  phis ${fixdir}/phis.nc                    gfs_data.tile7.halo0.nc_b
+  ${NCKS} -A -v radar_tten ${fixdir}/radar_tten_input.nc   gfs_data.tile7.halo0.nc_b
+
+  cp_vrfy ${bkpath}/sfc_data.tile7.halo0.nc        fv3_sfcdata
+  cp_vrfy gfs_data.tile7.halo0.nc_b                fv3_dynvars
+  ln_vrfy -s fv3_dynvars                           fv3_tracer
+
+  fv3sar_bg_type=1
+else
+  if [ ${DA_CYCLE_INTERV} -eq ${FCST_LEN_HRS} ]; then
+    restart_prefix=""
+  elif [ ${DA_CYCLE_INTERV} -lt ${FCST_LEN_HRS} ]; then
+    restart_prefix=${YYYYMMDD}.${HH}0000
+  else
+    print_err_msg_exit "\
+Restart hour should not larger than forecast hour:
+    Restart Hour = \"${DA_CYCLE_INTERV}\"
+    Forecast Hour = \"${FCST_LEN_HRS}\""    
+    exit 1
+  fi
+
+  cp_vrfy  ${bkpath}/${restart_prefix}.fv_core.res.tile1.nc             fv3_dynvars
+  cp_vrfy  ${bkpath}/${restart_prefix}.fv_tracer.res.tile1.nc           fv3_tracer
+  cp_vrfy  ${bkpath}/${restart_prefix}.sfc_data.nc                      fv3_sfcdata
+  ${NCKS} -A -v radar_tten ${fixdir}/radar_tten_restart.nc              fv3_tracer
+  fv3sar_bg_type=0
+fi
+
+# analysis time
```

Comment:
This comment doesn't help with understanding what's happening.

Reply:
Change to "update times in coupler.res to current cycle time"